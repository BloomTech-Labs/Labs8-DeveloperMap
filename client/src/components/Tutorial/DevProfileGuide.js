import React from 'react';
import styled from 'styled-components';
import seeker from '../../images/markerlogo.png';

class DevProfileGuide extends React.Component{
    switchPage = () => {
        this.props.history.push('/tutorial/comprofileguide');
    }
    render() {
    return (
        <ProfileGuide>
           <h1>Job Seeker Profile</h1>
           <p>
               By clicking on a <span><img src={seeker} alt="" style={{width:'50px'}}/></span> marker 
               on the map, you will be brought to that job seekers profile. The icons on the right will 
               bring you to various accounts for the user such as linkedIn or their portfolio page.
           </p>
           <h1>Picture goes here</h1>
           <p>
               At the bottom you will see their contact info if you are a verified company through MappaJob. 
           </p>
           <h1>Picture goes here</h1>
           <button onClick={this.switchPage}>Next</button>
        </ProfileGuide>
    );
    }
}

const ProfileGuide = styled.div`
  padding: 5%;
`;

export default DevProfileGuide;