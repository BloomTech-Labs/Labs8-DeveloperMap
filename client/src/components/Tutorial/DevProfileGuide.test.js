import React from 'react';
import renderer from 'react-test-renderer';
import DevProfileGuide from './DevProfileGuide';
import { shallow } from 'enzyme';

describe(`dev profile guide Component`, () => {
    it('renders correctly', () => {
        const tree = shallow(
          <DevProfileGuide />
        );
        expect(tree).toMatchSnapshot();
      });

});