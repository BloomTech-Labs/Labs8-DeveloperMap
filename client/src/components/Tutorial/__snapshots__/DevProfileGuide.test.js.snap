// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dev profile guide Component renders correctly 1`] = `
<Styled(styled.div)>
  <h1>
    Job Seeker Profile
  </h1>
  <p>
    By clicking on a
     
    <span>
      <img
        alt=""
        src="markerlogo.png"
        style={
          Object {
            "width": "50px",
          }
        }
      />
    </span>
     
    marker on the map, you will be brought to that job seekers profile. The icons on the right will bring you to various accounts for the user such as linkedIn or their portfolio page.
  </p>
  <img
    alt=""
    src="links.PNG"
  />
  <p>
    At the bottom you will see their contact info if you are a verified company through MappaJob.
  </p>
  <img
    alt=""
    src="contact.PNG"
  />
  <styled.button
    onClick={[Function]}
  >
    Next
  </styled.button>
</Styled(styled.div)>
`;
